RuriLib/Models/Blocks/Custom/ScriptBlockInstance.cs

```diff
@@ -174,24 +175,28 @@ public override string ToCSharp(List<string> definedVariables, ConfigSettings se
                     break;
 
                 case Interpreter.NodeJS:
+                    // for @Kermit with <3
```

Comment:
Please remove this, it's not very professional. I understand you did it for Kermit, but you can keep it in the PR's title / commit messages without littering the code 😅